docs/guides/insar_product_guide.md

```diff
@@ -0,0 +1,304 @@
+
+- The platform names, either Sentinel-1A or Sentinel-1B, are abbreviated "A" or "B", indicating the reference and secondary granule's imaging platform
+- The reference start date and time and the secondary start date and time, with the date and time separated by the letter T
+- The polarizations for the pair, either HH or VV, the orbit type, and the days of separation for the pair
+- The product type (always INT for InSAR) and the pixel spacing, which will be either 80 or 40, based upon the number of looks selected when the job was submitted for processing
+- The software package used for processing is always GAMMA for GAMMA InSAR products
+- User-defined options are denoted by three characters indicating whether the product is water masked (w) or not (u), the scene is clipped (e for entire area, c for clipped), and whether a single sub-swath was processed or the entire granule (either 1, 2, 3, or F for full swath)
+    - *Currently only the water masking is available as a user-selected option; the products always include the full granule extent with all three sub-swaths*
+- The filename ends with the ASF product ID, a 4 digit hexadecimal number
+
+
+![Figure 4](../images/asf_insar_names.png "Breakdown of ASF InSAR Naming Scheme")
+
+*Figure 4: Breakdown of ASF InSAR naming scheme.*
+
+### Image Files
+
+All of the main InSAR product files are 32-bit floating-point single-band GeoTIFFs.
+
+- The *amplitude* image is the calibrated radiometric backscatter from the reference granule in sigma-nought power. The image is terrain corrected using a geometric correction, but not radiometrically corrected. 
+- The *coherence* file pixel values range from 0.0 to 1.0, with 0.0 being completely non-coherent and 1.0 being perfectly coherent. 
+- The *unwrapped phase* file shows the results of the phase unwrapping process. This is the main interferogram output.
+- The *wrapped phase* file indicates the interferogram phase after applying the adpaptive filter immediately before unwrapping. *(optional)*
+- The *line-of-sight displacement* file indicates the displacement in meters along the look direction of the sensor. *(optional)*
+- The *vertical displacement* is generated from the line of sight displacement values, and makes the assumption that deformation only occurs in the vertical direction. Note that this assumption may not hold true in cases where the deformation also has a horizontal component.
+- The *look vectors* &#966 and &#952 describe the elevation and orientation angles of the sensor's look direction. *(optional)*
+- The *incidence angle* map gives the local incidence angle of the terrain. *(optional)*
+- The *water mask* image has pixels with a value of 0 for water and 1 for land. *(optional)*
+- The *DEM* file gives the local terrain heights in meters, with a geoid correction applied. *(optional)*
```

Comment:
Are these both also 32-bit floating point GeoTIFFs?

Reply:
The DEM is float32.  However, the water mask is byte.

Reply:
Addressed in PR #98